corehq/apps/app_manager/views/releases.py

```diff
@@ -230,8 +234,10 @@ def current_app_version(request, domain, app_id):
 def release_build(request, domain, app_id, saved_app_id):
     is_released = request.POST.get('is_released') == 'true'
     if not is_released:
-        if LatestEnabledBuildProfiles.objects.filter(build_id=saved_app_id).exists():
-            return json_response({'error': _('Please disable any enabled profiles to un-release this build.')})
+        if (LatestEnabledBuildProfiles.objects.filter(build_id=saved_app_id).exists() or
+                AppReleaseByLocation.objects.filter(build_id=saved_app_id, active=True).exists()):
+            return json_response({'error': _('Please disable any enabled profiles/location restriction '
```

Comment:
Didn't notice this earlier: does this error message actually get displayed to the user?

Reply:
Ya it was added after your last review. This things were suggested recently.
Yes, this get displayed to the user in browser prompt..

Reply:
Cool. Browser popups are not totally ideal, but they're quick to implement, so that seems fine.

Reply:
Agreed. I see that i added this last time considering how rarely this would be used and didn't pay much effort to it. Will revisit that.